.github/workflows/buf-ci.yaml

```diff
@@ -0,0 +1,22 @@
+name: Buf CI
+on:
+  push:
+    paths:
+      - 'proto/**'
+  pull_request:
+    paths:
+      - 'proto/**'
+    types: [opened, synchronize, reopened, labeled, unlabeled]
+  delete:
+permissions:
+  contents: read
+  pull-requests: write
+jobs:
+  buf:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: bufbuild/buf-action@v0.1
+        with:
+          version: "1.32.2"
```

Comment:
Do we need to set this version everywhere? For `buf`, it seems like overkill and might just be more maintenance work.

Reply:
I am personally fine dropping the explicit version here and always using the latest.

Reply:
Dropped version.